### ISSUE TITLE
[question] should we trim long strings in blog post titles?

### ISSUE THREAD

**Issue by pkra** [mathblogging/mathblogging-base#36]
If a blog post title contains very string (e.g., http://mindfuckmath.com/post/134753480836/99999999999999999999999-99999999999999999999999), then mathblogging.org's layout fails to fit into the viewport. 

We could think about adding `wbr` tags but I'm not sure if it's worth the trouble.

**Comment by scoskey** [comment]
Can't CSS overflow handle it?

**Comment by pkra** [comment]
> Can't CSS overflow handle it?

Yes. But theoretically blogs could post extremely long (think: document long) titles; makes me somewhat uncomfortable.

This might be useful https://github.com/bevacqua/trunc-html
